release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -132,10 +132,14 @@ This installation method is an alternative to uploading an {op-system} image on
 Before you set a path value for the `template` parameter, ensure that the default {op-system} boot image in the {product-title} release matches the {op-system} image template or virtual machine version; otherwise, cluster installation might fail.
 ====
 
+[id="ocp-4-14-OCP-on-ARM"]
+==== {product-title} on 64-bit ARM
+
+{product-title} {product-version} is now supported on 64-bit ARM architecture-based Google Cloud Platform installer-provisioned and user-provisioned infrastructures. You can also now use the `oc mirror` CLI plug in disconnected environments on 64-bit ARM clusters. For more information about instance availability and installation documentation, see xref:../installing/installing-preparing.adoc#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
```

Comment:
```suggestion
{product-title} {product-version} is now supported on 64-bit ARM architecture-based Google Cloud Platform installer-provisioned and user-provisioned infrastructures. You can also now use the `oc mirror` CLI plug-in disconnected environments on 64-bit ARM clusters. For more information about instance availability and installation documentation, see xref:../installing/installing-preparing.adoc#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p

Reply:
Nice catch! Thank you!